coverage_point_calculator/src/hexes.rs

```diff
@@ -5,13 +5,40 @@ use rust_decimal_macros::dec;
 
 use crate::{BoostedHexStatus, RadioType, Result};
 
+/// Breakdown of points for a hex.
+///
+/// Example:
+///   Outdoor Wifi with 1-hex boosted 5x,
+///   Rank 2, Assignment AAA:
+///     SplitPoints {
+///       modeled: 16,
+///       base: 8,
+///       boosted: 32
+///     }
+///
+///   Rank 2 splits modeled points in half.
+///   Boost at 5x adds 32 points ( (8 * 5) - 8 )
+#[derive(Debug, Default, Clone)]
+pub struct HexPoints {
+    /// Default points received for hex
+    ///
+    /// (RadioType, SignalLevel) points
+    pub(crate) modeled: Decimal,
+    /// Points including Coverage affected multipliers
+    ///
+    /// modeled + (Rank * Assignment)
+    pub(crate) base: Decimal,
+    /// Points _over_ normal received from hex boosting.
+    ///
+    /// (base * Boost multiplier) - base
+    pub(crate) boosted: Decimal,
+}
+
 #[derive(Debug, Clone)]
 pub struct CoveredHex {
     pub hex: hextree::Cell,
-    /// Default points received from (RadioType, SignalLevel) pair.
-    pub base_coverage_points: Decimal,
```

Comment:
Removing this field breaks a doc comment link at `lib.rs:8`

Reply:
Fixed, thank you. Still working on making `cargo doc` part of my muscle memory.